app.js

```diff
@@ -258,7 +265,7 @@ app.get("/login", function (req, res, next) {
     if (_.isString(req.session.relayState)) {
       req.query.RelayState = req.session.relayState;
     }
-    spOptions.forceAuthn = (req.query.forceauthn !== undefined);
+    //spOptions.forceAuthn = (req.query.forceauthn !== undefined); go based off the parameter on startup?
```

Comment:
Why configure in the settings instead of specify in the query parameter? It seems it is more flexible to just call /login?forceauthn=true